web/src/main/java/com/epam/esm/controllers/GiftCertificateController.java

```diff
@@ -0,0 +1,127 @@
+package com.epam.esm.controllers;
+
+
+import com.epam.esm.dto.GiftCertificateDto;
+import com.epam.esm.entities.GiftCertificate;
+import com.epam.esm.entities.Tag;
+import com.epam.esm.exception.InvalidFieldException;
+import com.epam.esm.exception.ResourceNotFoundException;
+import com.epam.esm.service.GiftCertificateService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+/**
+ *  The class that represents an API for basic giftCertificate-related application operations
+ *  @author Yauheni Vezhnavets
+ **/
+
+
+@RestController
+@RequestMapping("/giftCertificates")
```

Comment:
please follow best practices in Rest naming conventions https://nordicapis.com/10-best-practices-for-naming-api-endpoints/

Reply:
Changed for @RestController
@RequestMapping("/certificates")